padl/transforms.py

```diff
@@ -405,12 +405,16 @@ def pd_save(self, path: Union[Path, str], force_overwrite: bool = False,
         if path.suffix == '':
             path = path.parent / (path.name + '.padl')
 
-        if path.exists():
+        if path.exists() and list(path.glob('*')):
             if not force_overwrite:
                 raise FileExistsError(f'{path} exists, call with *force_overwrite* to overwrite.')
-            rmtree(path)
 
-        path.mkdir()
+            with TemporaryDirectory('.padl') as dirname:
+                self.pd_save(dirname, False)
+                rmtree(path)
+                copytree(dirname, path)
```

Comment:
I think it could be a good idea to use a `move` method instead to avoid the situation where two copies of the weights exist at one time. This potentially could be a problem when working with large models. What do you think?

Reply:
Never mind I don't think this will be a practical problem people have and I don't think it's worth changing.